im.bernard.Memorado.json

```diff
@@ -0,0 +1,51 @@
+{
+    "id" : "im.bernard.Memorado",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "memorado",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
```

Comment:
this need the cleanup rule to remove it after build.

Reply:
See https://github.com/flathub/re.sonny.Tangram/blob/master/re.sonny.Tangram.json#L33

Reply:
Should be fixed now